.travis.yml

```diff
@@ -14,11 +13,12 @@ cache:
 
 env:
   - NPM_SCRIPT=node-test
-  - NPM_SCRIPT=client-test
 
 matrix:
   fast_finish: true
   include:
+  - node_js: "4.2"
+    env: NPM_SCRIPT=client-test
   - node_js: "5"
```

Comment:
Lets swap this for 6?  6 is the current stable (as it is we aren't testing current stable at all), and will become an LTS soon (technically 2016-10-01 but I think there will need to be a release first).